docs/UnderstandingCLI.md

```diff
@@ -0,0 +1,38 @@
+---
+id: understanding-cli
+title: Understanding CLI
+layout: docs
+category: Guides
+permalink: docs/understanding-cli.html
+next: upgrading
+previous: performance
+---
+
+Though you may have installed the `react-native-cli` via npm as a separate module, it is a shell for accessing the CLI embedded
+in the React Native of each project. Your commands and their effects are dependent on the version of the module of `react-native` 
+in context of the project. This guide will give a brief overview of the CLI in the module.
+
+# Location and import of the local CLI
+
+React Native has a [`local-cli`](https://github.com/facebook/react-native/tree/master/local-cli) folder with a file named 
+[`cliEntry.js`](https://github.com/facebook/react-native/blob/master/local-cli/cliEntry.js).  Here, the commands are read 
+from `commands.js` and added as possible CLI commands.  _E.G._ the `react-native link` command, existis in the 
+[`react-native/local-cli/link`](https://github.com/facebook/react-native/blob/master/local-cli/link/link.js) folder, and is
+imported/required in `commands.js`, which will register it as a documented command to be exposed to the CLI.
+
+# Command protocol

```

Comment:
How about 'Command definitions'? It's not really a protocol in the sense of a communication protocol.